pkg/apis/ext.cattle.io/v1/types.go

```diff
@@ -20,10 +20,21 @@ type UserActivity struct {
 }
 
 type UserActivitySpec struct {
+	// TokenId is the token Id for which the UserActivity will update
+	// the LastIdleTimeout value on the Token resource.
+	// +optional
 	TokenId string `json:"tokenId"`
```

Comment:
```suggestion
	TokenID string `json:"tokenId"`
```

Reply:
Note: Applying this suggestion will require a slight change to the preceding comment as well.

Reply:
yes, makes sense, I'll solve all the conflicts generated by this suggestion. Thanks for the advice.